### PR TITLE
main/openrc: use tod as hwclock on s390x

### PR DESCRIPTION
**Pull request by tuan-hoang1**

### CODE REVIEW

File: main/openrc/use-tod-as-clocksource-for-s390x.patch [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
++	: $(( retval += $? ))
+ 
+ 	eend $retval "Failed to set the system clock"
+ 
+@@ -131,7 +133,7 @@ start()
+ stop()
+ {
+ 	# Don't tweak the hardware clock on LiveCD halt.
+-	[ -n "$CDBOOT" ] && return 0
++	[ -n "$CDBOOT" ] || [ "$(uname -m)" = "s390x" ] && return 0
```
Comment (ncopa) [inline main/openrc/use-tod-as-clocksource-for-s390x.patch]: Can we change this to an `if ...; then ...` statement instead? Its more readable.

**Comment by ncopa** [comment]
I think this should be suggested for upstream openrc. https://github.com/OpenRC/openrc

**Comment by tuan-hoang1** [comment]
Okay. Will do. Thanks

**Comment by tuan-hoang1** [comment]
As I comment in openrc thread, I just figured out s390x needs neither hwclock nor swclock.

hwclock is enabled already and it returns immediately on s390x, which is harmless.

If the hwclock.init message is annoying enough, we will disable hwclock in the future :)
Closing.

**Pull request closed without merging**